### PR TITLE
Add Odil

### PR DESCRIPTION
**Pull request by lamyj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by lamyj** [comment]
@conda-forge/help-c-cpp: this recipe is mostly ready for review. The Windows build is painfully slow (about 4x slower than Linux or macOS) and thus times out on Appveyor; the Azure pipeline is however OK. To fix this, should I update the `provider: win` in `conda-forge.yml` in this pull request or is this something to be done once the `odil-feedstock` repo is created?

**Comment by scopatz** [comment]
You can ignore the Appveyor failures if Azure is running

File: recipes/odil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  skip: True  # [py<=35 or (win and vc<14)]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - boost
+    # - dcmtk
```
Comment (scopatz) [inline recipes/odil/meta.yaml]: Can you add a comment as to why this is commented out or remove it please?

Comment (lamyj) [reply recipes/odil/meta.yaml]: Done in a5c81313b, thanks.

**Comment by scopatz** [comment]
Awesome thanks!

**Pull request merged**